### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by MrMondrian**
## Related Issue(s)
  - [Issue #1](https://github.com/orgs/mcgill-robotics/projects/2?pane=issue&itemId=38115686)

## Other Dependent PRs
  - https://github.com/mcgill-robotics/AUV-2024/pull/273

## Changes Made
- fixes bug in no bypass. add launch file for actions test

## Tests Done
- Tested locally in container

### CODE REVIEW

**Comment by MrMondrian** [comment]
@antoinedang @frompotenza Quick approvals would be good. I need this on the main branch to test the actions

**Comment by frompotenza** [comment]
is this just for docker?

**Review by frompotenza — APPROVED** [review]

**Comment by MrMondrian** [comment]
There’s also a no bypass change I forgot to merge in the last pr but otherwise yes it just adds basically a sim.launch clone that runs in server mode.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: src/bridge.py [hunk 1]
```diff
@@ -40,7 +40,8 @@ def cb_sim_dvl_depth(data):
     #DVL QUATERNION
     q_NWU_auv = np.quaternion(pose_NWU_auv.orientation.w, pose_NWU_auv.orientation.x, pose_NWU_auv.orientation.y, pose_NWU_auv.orientation.z)
     q_dvlref_dvl = q_NWU_dvlref.inverse() * q_NWU_auv * q_dvl_auv.inverse()
-    euler_dvlref_auv = quaternion.as_euler_angles(q_dvlref_dvl)
+    # euler_dvlref_auv = quaternion.as_euler_angles(q_dvlref_dvl)
```
Comment (antoinedang) [inline src/bridge.py:43]: why transformations instead of quaternion?

File: launch/test.launch
Comment (antoinedang) [inline launch/test.launch:1]: as i said in the other PR the name test.launch is not very clear, could we rename it to something more explicit

**Review by antoinedang — CHANGES_REQUESTED** [review]
commented

File: src/bridge.py (hunk 1 above)
Comment (MrMondrian) [reply src/bridge.py:43]: its what vasily was using. idk what order the euler angles are for the dvl, just that the default mode for the transformations library is correct

File: launch/test.launch
Comment (MrMondrian) [reply launch/test.launch:1]: fixed

**Comment by MrMondrian** [comment]
@antoinedang re-requesting your review. This needs to be on the main branch to test it for actions. It works locally and when run manually in the container

**Review by antoinedang — APPROVED** [review]

**Pull request merged**